### PR TITLE
[Py] Added unsqueeze operator to Python API

### PR DESCRIPTION
**Pull request by Ewa21**

### CODE REVIEW

File: doc/sphinx/source/python_api/_autosummary/ngraph.ops.rst [hunk 1]
```diff
@@ -77,6 +77,7 @@ ngraph.ops
       tan
       tanh
       topk
+      unsqueeze
```
Comment (tsocha) [inline doc/sphinx/source/python_api/_autosummary/ngraph.ops.rst]: I know this is not your fault.
But do we really need to many empty lines at the end of this file?

File: python/ngraph/__init__.py [hunk 2]
```diff
@@ -90,5 +90,7 @@
 from ngraph.ops import tan
 from ngraph.ops import tanh
 from ngraph.ops import topk
+from ngraph.ops import unsqueeze
+
```
Comment (tsocha) [inline python/ngraph/__init__.py]: Is this empty line necessary?

**Comment by postrational** [comment]
TEST NOW

**Comment by postrational** [comment]
TESTNOW

**Pull request merged**